starters/svelte-kit-scss/src/lib/components/Counter.svelte

```diff
@@ -1,101 +1,48 @@
 <script lang="ts">
-  import { spring } from 'svelte/motion';
+  import { countStore } from '$lib/stores';
 
-  let count = 0;
+  const reset = (): void => countStore.set(0);
+  const increment = (): void => countStore.update((x) => x + 1);
+  const decrement = (): void => countStore.update((x) => x - 1);
 
-  const displayed_count = spring();
-  $: displayed_count.set(count);
-  $: offset = modulo($displayed_count, 1);
+  let count: number;
 
-  function modulo(n: number, m: number) {
-    // handle negative numbers
-    return ((n % m) + m) % m;
-  }
+  countStore.subscribe((value) => {
+    count = value;
+  });
 </script>
 
-<div class="counter">
-  <button on:click={() => (count -= 1)} aria-label="Decrease the counter by one">
-    <svg aria-hidden="true" viewBox="0 0 1 1">
-      <path d="M0,0.5 L1,0.5" />
-    </svg>
-  </button>
-
-  <div class="counter-viewport">
-    <div class="counter-digits" style="transform: translate(0, {100 * offset}%)">
-      <strong class="hidden" aria-hidden="true">{Math.floor($displayed_count + 1)}</strong>
-      <strong>{Math.floor($displayed_count)}</strong>
-    </div>
-  </div>
-
-  <button on:click={() => (count += 1)} aria-label="Increase the counter by one">
-    <svg aria-hidden="true" viewBox="0 0 1 1">
-      <path d="M0,0.5 L1,0.5 M0.5,0 L0.5,1" />
-    </svg>
-  </button>
+<div class="container">
+  <span class="result">Count: {count}</span>
```

Comment:
you can use autosubscription, to make sure you are unsubscribed as well.
This is also a reason why i suggest to name it  `count`  instead of `countStore`
```suggestion
  <span class="result">Count: {$countStore}</span>
```